python-package/lightgbm/basic.py

```diff
@@ -2261,9 +2261,11 @@ def predict(self, data, num_iteration=None,
 
             Note
             ----
-            If you want to get more explanation for your model's predictions using SHAP values
+            If you want to get more explanations for your model's predictions using SHAP values
             like SHAP interaction values,
-            you can install shap package (https://github.com/slundberg/shap).
+            you can install the shap package (https://github.com/slundberg/shap). Note that unlike
```

Comment:
Please, start new sentence from a new line. This makes docstrings more readable.

Reply:
While we're at it, extremely minor format inconsistency in 

> shape = [n_samples] or shape = [n_samples, n_classes]

vs 

> shape = [n_samples, n_trees] or shape [n_samples, n_trees * n_classes]

Slightly OCD, I'll admit

Reply:
@maximilianeber could you elaborate what you mean there?

Reply:
@maximemerabet

Reply:
@maximilianeber Do you mean sign `=` after shape word?

Reply:
@maximilianeber sorry for the auto-complete fail :) ...@maximemerabet just fixed the equals

Reply:
Haha.

Yes was referring to the equals!